test/spec/practiceSpec.js

```diff
@@ -1,122 +1,124 @@
-			var arr = [1, 2, 3];
-			looper(arr);
-			expect(alert.calls.count()).toBe(arr.length);
-		})
-		it('should call the items in order from first to last', function () {
-			var alert = spyOn(window, 'alert');
-			var arr = [1, 2, 3];
-			looper(arr);
-			for (var i = 0; i < arr.length; i++) {
-				expect(alert.calls.argsFor(i)[0]).toBe(arr[i]);
-			}
-		})
-	})
-	describe('reversedLooper', function () {
-		it('should exist', function () {
-			expect(reversedLooper).toBeDefined();
-		})
-		it('should call alert once for each item in the array', function () {
-			var alert = spyOn(window, 'alert');
-			var arr = [1,2,3];
-			reversedLooper(arr);
-			expect(alert.calls.count()).toBe(arr.length);
-		})
-		it('should call the items in order from last to first', function () {
-			var alert = spyOn(window, 'alert');
-			var arr = [1, 2, 3];
-			reversedLooper(arr);
-			for (var i = 0; i < arr.length; i++) {
-				expect(alert.calls.argsFor(i)[0]).toBe(arr[arr.length - 1 - i]);
-			}
-		})
-	})
-	describe('evenFinder', function () {
-		it('should exist', function () {
-			expect(evenFinder).toBeDefined();
-		})
-		it('should be a function', function () {
-			expect(evenFinder).toEqual(jasmine.any(Function));
-		})
-		it('should return an array', function () {
-			expect(evenFinder([])).toEqual(jasmine.any(Array));
-		})
-		it('should return an array with only even numbers', function () {
-			var arr = [1, 2, 3, 4, 5, 6];
-			var evens = evenFinder(arr);
-			expect(evens).toEqual([2, 4, 6]);
-		})
-	})
-	describe('divider', function () {
-		it('should exist', function () {
-			expect(divider).toBeDefined();
-		})
-		it('should be a function', function () {
-			expect(divider).toEqual(jasmine.any(Function));
-		})
-		it('should return an array of two arrays', function () {
-			expect(divider([])).toEqual([jasmine.any(Array), jasmine.any(Array)])
-		})
-		it('should have even numbers in the first array and odd numbers in the second array', function () {
-			var arr = [1, 2, 3, 4, 5, 6, 7, 8, 9];
-			var divided = divider(arr);
-			var expected = [[2, 4, 6, 8], [1, 3, 5, 7, 9]]
-			expect(divided).toEqual(expected);
-		})
-	})
-	describe('finder', function () {
-		it('should exist', function () {
-			expect(finder).toBeDefined();
-		})
-		it('should be a function', function () {
-			expect(finder).toEqual(jasmine.any(Function));
-		})
-		it('should return a boolean', function () {
-			expect(finder([])).toEqual(jasmine.any(Boolean));
-		})
-		it('should return true if the random number is found in the array', function () {
-			var rand = spyOn(window, 'getRandomArbitrary').and.returnValue(3);
-			var arr = [1, 2, 3];
-			expect(finder(arr)).toBe(true);
-		})
-		it('should return false if the random number is not found in the array', function () {
-			var rand = spyOn(window, 'getRandomArbitrary').and.returnValue(4);
-			var arr = [1, 2, 3];
-			expect(finder(arr)).toBe(false);
-		})
-	})
+	// describe('first', function () {
```

Comment:
Why are we commenting all this out?

Reply:
That was commented out in a previous edit. I'm not sure to the reasoning.

Reply:
Can you look at see if it's still used and either comment it back in, or remove it entirely.